periodic sync upstream KF to midstream ODH

Al-Pragliola opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
<!--- [UI] Include any screenshots of changed UI; Include any gifs if it was a flow / UX change -->

- keep https://github.com/kubeflow/model-registry/pull/960
- keep https://github.com/kubeflow/model-registry/pull/1033
- keep https://github.com/kubeflow/model-registry/pull/1068
- keep https://github.com/kubeflow/model-registry/pull/1062
- keep https://github.com/kubeflow/model-registry/pull/1065
- keep https://github.com/kubeflow/model-registry/pull/1066
- keep https://github.com/kubeflow/model-registry/pull/1063
- keep https://github.com/kubeflow/model-registry/pull/1069
- keep https://github.com/kubeflow/model-registry/pull/1070
- keep https://github.com/kubeflow/model-registry/pull/1058
- keep https://github.com/kubeflow/model-registry/pull/1073
- keep https://github.com/kubeflow/model-registry/pull/1077
- keep https://github.com/kubeflow/model-registry/pull/1053
- keep https://github.com/kubeflow/model-registry/pull/1054
- keep https://github.com/kubeflow/model-registry/pull/1056
- keep https://github.com/kubeflow/model-registry/pull/1072
- keep https://github.com/kubeflow/model-registry/pull/986
- keep https://github.com/kubeflow/model-registry/pull/992
- keep https://github.com/kubeflow/model-registry/pull/1087
- keep https://github.com/kubeflow/model-registry/pull/1076
- keep https://github.com/kubeflow/model-registry/pull/1089
- keep https://github.com/kubeflow/model-registry/pull/993
- keep https://github.com/kubeflow/model-registry/pull/1086
- keep https://github.com/kubeflow/model-registry/pull/1093
- keep https://github.com/kubeflow/model-registry/pull/1099
- keep https://github.com/kubeflow/model-registry/pull/1100
- keep https://github.com/kubeflow/model-registry/pull/1101
- keep https://github.com/kubeflow/model-registry/pull/1102
- keep https://github.com/kubeflow/model-registry/pull/1103
- keep https://github.com/kubeflow/model-registry/pull/1091
- keep https://github.com/kubeflow/model-registry/pull/1095
- keep https://github.com/kubeflow/model-registry/pull/1090
- keep https://github.com/kubeflow/model-registry/pull/1105
- keep https://github.com/kubeflow/model-registry/pull/1106
- keep https://github.com/kubeflow/model-registry/pull/1094
- keep https://github.com/kubeflow/model-registry/pull/1096
- keep https://github.com/kubeflow/model-registry/pull/1097
- keep https://github.com/kubeflow/model-registry/pull/1098
- keep https://github.com/kubeflow/model-registry/pull/1104
- keep https://github.com/kubeflow/model-registry/pull/1112
- keep https://github.com/kubeflow/model-registry/pull/1109
- keep https://github.com/kubeflow/model-registry/pull/1074
- keep https://github.com/kubeflow/model-registry/pull/1088
- keep https://github.com/kubeflow/model-registry/pull/1110
- (merge commit)

labels are applied,

superseeds https://github.com/opendatahub-io/model-registry/pull/198

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Merge criteria:
<!--- This PR will be merged by any repository approver when it meets all the points in the checklist -->
- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [ ] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work.
- [ ] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).
- [ ] **For first time contributors**: Please reach out to the [Reviewers](../OWNERS) to ensure all tests are being run, ensuring the label `ok-to-test` has been added to the PR.

If you have UI changes

<!--- You can ignore these if you are doing Go Model Registry REST server, MR Python client, manifest, controller, internal logic, etc changes; aka non-UI / visual changes -->
- [ ] The developer has added tests or explained why testing cannot be added.
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Verify that UI/UX changes conform the UX guidelines for Kubeflow.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Added a workflow to identify and assist first-time contributors on pull requests.
  - Introduced a theme-aware search input and form group wrapper for a unified UI experience across themes.
  - Added roadmap documentation outlining project milestones and future plans.

- **Enhancements**
  - Unified and simplified UI components by removing theme-based conditional rendering, resulting in consistent layouts and interactions.
  - Improved error messaging and logging in backend and middleware for clearer diagnostics.
  - Upgraded Go, Python, and frontend dependencies for better performance and compatibility.
  - Updated documentation with clearer installation instructions, improved contributor guidelines, and expanded adopter listings.

- **Bug Fixes**
  - Corrected accessor methods in API models to ensure field accessors match their intended fields.

- **Refactor**
  - Streamlined OpenAPI schemas and removed redundant intermediate models for a simpler and more maintainable API.
  - Reorganized and cleaned up code and configuration files for improved readability and maintainability.
  - Enhanced test coverage and reliability for model registration and InferenceService synchronization.

- **Chores**
  - Updated CI workflows and Dockerfiles to use newer dependency versions and modern best practices.
  - Removed deprecated or unused files and code, including OpenAPI-generated base models.

- **Tests**
  - Added new end-to-end and unit tests for model registration, S3 data connection, and InferenceService URL synchronization.

- **Style**
  - Improved formatting and styling in documentation and UI for better clarity and user experience.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

spolti commented:
A small change that I do when something fails on apple, is to force podman to use amd64 in the build commands.
podman build --arch=amd64.

works pretty nice.

We could have in the Makefile a if to append args, e.g. when using podman && arm, appends args.

tarilabs commented:
Ciao @spolti !! 👋 

I've tried Podman, but for some reasons it didn't work well on Mac (at least I've installed all necessary Mac-helpers) specifically when using TestContainers for Go it required code-changes (beyond config changes) unfortunately.
It's a bit sad because I really preferred to use Podman if possible.

I'm not against using it, but at least when developing for this project, so far the best experience was with Colima.

Always happy to hear if anyone can report difference experiences!

spolti commented:
Thanks for the explanation @tarilabs, how did you install podman?
I didn't install any helper, it works ok (also exposes the podman registry for docker so docker understand that it does have a registry).

All I did was install podman and podman-desktop.

spolti commented:
For that line in specific, maybe not hardcode it?
Instead, use a env, and default to that value if the given env is not set :)

tarilabs commented:
> Instead, use a env, and default to that value if the given env is not set :)

that is clearly an option, but somehow according to Testcontainers that should not even be needed; unfortunately I didn't have bandwidth to look further into this, given using alternative docker-engine managers just worked

spolti commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,181 @@
+...
+```
+
+and now you can substitute `gmake` every time the make command is mentioned in guides (or perform the path management per the caveat).
````
isn't better to use just make as stated in the comment? :)

tarilabs replied:
you mean to adopt the caveat instructions per homebrew install log? :)
that is optional, so I want to keep it same options for our readers (personally I'm just using gmake)

isinyaaa commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,181 @@
+The experience is very similar to when on GitHub.com you press dot `.` and you get a VSCode "inside something", except it is local to your machine.
+It's not super fast because x86 is emulated via Rosetta, but works "good enough" to complete most tasks without requiring remote connection to a real x86 server.
+
+For non-VSCode user but leveraging LSPs, an SDK is available as well on the upstream: https://containers.dev.
```
From their docs these editors are supported: https://containers.dev/supporting.html#editors

For my use-case (neovim user) it isn't possible to integrate unless I use a custom extension. Either way, this doesn't have to do with LSPs IIUC.

tarilabs replied:
done with  a127c05

Al-Pragliola committed:
feat(controller): added reconciliation of inferenceservice url (#960)

* feat(controller): added reconciliation to inferenceservice url

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

* fix(controller): remove FIt from tests

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

* fix(controller): linting issues

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

* chore(controller): lower complexity of customproperties check

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Alessio Pragliola <83355398+Al-Pragliola@users.noreply.github.com>

* fix(controller): potential nil pointer exception

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

* chore(controller): remove sync.Maps as are not needed

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

* feat(controller): improve test coverage

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>

---------

Signed-off-by: Alessio Pragliola <seth.pro@gmail.com>
Signed-off-by: Alessio Pragliola <83355398+Al-Pragliola@users.noreply.github.com>
Co-authored-by: Paul Boyd <paul@camelot.email>

dbasunag committed:
specify python version range for ray and add importorskip for ray in … (#1033)

* specify python version range for ray and add importorskip for ray in test

Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

* Update clients/python/pyproject.toml

Co-authored-by: Matteo Mortari <matteo.mortari@gmail.com>
Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

---------

Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>
Co-authored-by: Matteo Mortari <matteo.mortari@gmail.com>

milosjava committed:
add company to model registry adopters (#1068)

* add company to model registry adopters

Signed-off-by: milosjava <milosjava@gmail.com>

* add more details and fixed formatting

Signed-off-by: milosjava <milosjava@gmail.com>

---------

Signed-off-by: milosjava <milosjava@gmail.com>

dependabot[bot] committed:
build(deps-dev): bump ray from 2.44.1 to 2.45.0 in /clients/python (#1062)

Bumps [ray](https://github.com/ray-project/ray) from 2.44.1 to 2.45.0.
- [Release notes](https://github.com/ray-project/ray/releases)
- [Commits](https://github.com/ray-project/ray/compare/ray-2.44.1...ray-2.45.0)

---
updated-dependencies:
- dependency-name: ray
  dependency-version: 2.45.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump ruff from 0.11.7 to 0.11.8 in /clients/python (#1065)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.11.7 to 0.11.8.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.11.7...0.11.8)

---
updated-dependencies:
- dependency-name: ruff
  dependency-version: 0.11.8
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump boto3 from 1.38.3 to 1.38.8 in /clients/python (#1066)

Bumps [boto3](https://github.com/boto/boto3) from 1.38.3 to 1.38.8.
- [Release notes](https://github.com/boto/boto3/releases)
- [Commits](https://github.com/boto/boto3/compare/1.38.3...1.38.8)

---
updated-dependencies:
- dependency-name: boto3
  dependency-version: 1.38.8
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump pydantic from 2.11.3 to 2.11.4 in /clients/python (#1063)

Bumps [pydantic](https://github.com/pydantic/pydantic) from 2.11.3 to 2.11.4.
- [Release notes](https://github.com/pydantic/pydantic/releases)
- [Changelog](https://github.com/pydantic/pydantic/blob/main/HISTORY.md)
- [Commits](https://github.com/pydantic/pydantic/compare/v2.11.3...v2.11.4)

---
updated-dependencies:
- dependency-name: pydantic
  dependency-version: 2.11.4
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

rareddy committed:
Adding Red hat as one of the adopters of Kubeflow Model Registry (#1069)

Signed-off-by: Ramesh Reddy <rareddy@redhat.com>

tarilabs committed:
docs: fix typo (#1070)

* docs: fix typo

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Update RELEASE.md

Co-authored-by: Alessio Pragliola <83355398+Al-Pragliola@users.noreply.github.com>
Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Update RELEASE.md

Co-authored-by: Alessio Pragliola <83355398+Al-Pragliola@users.noreply.github.com>
Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

---------

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>
Co-authored-by: Alessio Pragliola <83355398+Al-Pragliola@users.noreply.github.com>

tarilabs committed:
docs(py): minor fix for the tutorial (#1058)

- in the previous section, the client is referred as `registry.method()`
- suggest to the reader the expected import, useful for copy-paste

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

ederign committed:
chore(bff): cleanup as a fup to #918 (#1073)

Signed-off-by: Eder Ignatowicz <ignatowicz@gmail.com>

mginfn committed:
chore: Update ADOPTERS.md (#1077)

Signed-off-by: mginfn <116359611+mginfn@users.noreply.github.com>

tarilabs committed:
gha: workflow for comment first-time PR, onboarding (#1053)

instead of just relying on self-assessed bulletpoint in PR template:
https://github.com/kubeflow/model-registry/blame/de5f225d96a4daeca77506d233082b1c4ea5afa3/.github/pull_request_template.md#L21

this adds a comment to help out with the review of the PR.

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

tarilabs committed:
doc(py): fix by move this section in the correct place (#1054)

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

tarilabs committed:
docs: add Blog about KF 1.10 entry (#1056)

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

ederign committed:
chore(bff): adding vscode/cursor debug capabilities on bff (#1072)

Signed-off-by: Eder Ignatowicz <ignatowicz@gmail.com>

dependabot[bot] committed:
build(deps): bump http-proxy-middleware from 2.0.7 to 2.0.9 in /clients/ui/frontend (#986)

Bumps [http-proxy-middleware](https://github.com/chimurai/http-proxy-middleware) from 2.0.7 to 2.0.9.
- [Release notes](https://github.com/chimurai/http-proxy-middleware/releases)
- [Changelog](https://github.com/chimurai/http-proxy-middleware/blob/v2.0.9/CHANGELOG.md)
- [Commits](https://github.com/chimurai/http-proxy-middleware/compare/v2.0.7...v2.0.9)

---
updated-dependencies:
- dependency-name: http-proxy-middleware
  dependency-version: 2.0.9
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

jenny-s51 committed:
Apply Iconography, Descriptions, and Color Palette to Page Sections (#992)

Signed-off-by: Jenny <32821331+jenny-s51@users.noreply.github.com>

fix tests

jiekechoo committed:
Update ADOPTERS.md (#1087)

Signed-off-by: Jieke Choo <jiekechoo@yiqisoft.cn>

ederign committed:
chore(deps): bump Go to 1.24.3 and update k8s/client dependencies (#1076)

- Upgrade Go version to 1.24.3 across gha, Dockerfile, go.mod
- Bump Kubernetes dependencies to v0.33.0
- Update golangci-lint action and version (v7 → v8, v2.0.2 → v2.1.0)
- Refresh go.sum with indirect dependency updates
- Minor README cleanup and formatting fixes

Signed-off-by: Eder Ignatowicz <ignatowicz@gmail.com>

ederign committed:
chore(frontend): update development dependencies (#1089)

- Update sass to v1.87.0
- Update TypeScript ESLint packages to v8.31.1
- Update ESLint plugins:
  - eslint-plugin-prettier to v5.4.0
  - eslint-plugin-react to v7.37.5

Signed-off-by: Eder Ignatowicz <ignatowicz@gmail.com>

jenny-s51 committed:
Refactor Theme-Dependent MUI Component Instances (#993)

Signed-off-by: Jenny <32821331+jenny-s51@users.noreply.github.com>

refactor instances of FormGroup and SearchInput

remove unnecessary comments

Signed-off-by: Jenny <32821331+jenny-s51@users.noreply.github.com>

remove unnecessary comments

tarilabs committed:
docs: refresh CONTRIBUTING.md (#1086)

* docs: refresh CONTRIBUTING.md

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Update CONTRIBUTING.md

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

---------

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>
Co-authored-by: Paul Boyd <paul@camelot.email>

tiansiyuan committed:
Add @tiansiyuan (#1093)

Signed-off-by: Tian Siyuan <tiansiyuan@gmail.com>

dependabot[bot] committed:
build(deps): bump olot from 0.1.6 to 0.1.7 in /clients/python (#1099)

Bumps olot from 0.1.6 to 0.1.7.

---
updated-dependencies:
- dependency-name: olot
  dependency-version: 0.1.7
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump boto3 from 1.38.8 to 1.38.13 in /clients/python (#1100)

Bumps [boto3](https://github.com/boto/boto3) from 1.38.8 to 1.38.13.
- [Release notes](https://github.com/boto/boto3/releases)
- [Commits](https://github.com/boto/boto3/compare/1.38.8...1.38.13)

---
updated-dependencies:
- dependency-name: boto3
  dependency-version: 1.38.13
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump ray from 2.45.0 to 2.46.0 in /clients/python (#1101)

Bumps [ray](https://github.com/ray-project/ray) from 2.45.0 to 2.46.0.
- [Release notes](https://github.com/ray-project/ray/releases)
- [Commits](https://github.com/ray-project/ray/compare/ray-2.45.0...ray-2.46.0)

---
updated-dependencies:
- dependency-name: ray
  dependency-version: 2.46.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump huggingface-hub from 0.30.2 to 0.31.1 in /clients/python (#1102)

Bumps [huggingface-hub](https://github.com/huggingface/huggingface_hub) from 0.30.2 to 0.31.1.
- [Release notes](https://github.com/huggingface/huggingface_hub/releases)
- [Commits](https://github.com/huggingface/huggingface_hub/compare/v0.30.2...v0.31.1)

---
updated-dependencies:
- dependency-name: huggingface-hub
  dependency-version: 0.31.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump ruff from 0.11.8 to 0.11.9 in /clients/python (#1103)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.11.8 to 0.11.9.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.11.8...0.11.9)

---
updated-dependencies:
- dependency-name: ruff
  dependency-version: 0.11.9
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tarilabs committed:
OWNERS: add ederign as approvers (#1091)

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

dependabot[bot] committed:
build(deps): bump github.com/golang/glog from 1.2.4 to 1.2.5 (#1095)

Bumps [github.com/golang/glog](https://github.com/golang/glog) from 1.2.4 to 1.2.5.
- [Release notes](https://github.com/golang/glog/releases)
- [Commits](https://github.com/golang/glog/compare/v1.2.4...v1.2.5)

---
updated-dependencies:
- dependency-name: github.com/golang/glog
  dependency-version: 1.2.5
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

rareddy committed:
API Simplification - Flatten base types (#1090)

* chore(openapi): flatten base types

Signed-off-by: Paul Boyd <pboyd@redhat.com>

* Simplification of the API of the Model Registry by reducing nested objects and making the API little flat

Signed-off-by: Ramesh Reddy <rareddy@redhat.com>

---------

Signed-off-by: Paul Boyd <pboyd@redhat.com>
Signed-off-by: Ramesh Reddy <rareddy@redhat.com>
Co-authored-by: Paul Boyd <pboyd@redhat.com>

lugi0 committed:
Move robot TC to pytest (#1105)

* Translate  robot TC to pytest

Signed-off-by: lugi0 <lgiorgi@redhat.com>

* Update test to only use methods exposed directly by the client

Signed-off-by: lugi0 <lgiorgi@redhat.com>

---------

Signed-off-by: lugi0 <lgiorgi@redhat.com>

tarilabs committed:
py: change str repr to mitigate confusion (#1106)

* py: change str repr to mitigate confusion

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* linting

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

---------

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

dependabot[bot] committed:
build(deps): bump github.com/kubeflow/model-registry from 0.2.14 to 0.2.17 in /clients/ui/bff (#1094)

Bumps [github.com/kubeflow/model-registry](https://github.com/kubeflow/model-registry) from 0.2.14 to 0.2.17.
- [Release notes](https://github.com/kubeflow/model-registry/releases)
- [Changelog](https://github.com/kubeflow/model-registry/blob/main/RELEASE.md)
- [Commits](https://github.com/kubeflow/model-registry/compare/v0.2.14...v0.2.17)

---
updated-dependencies:
- dependency-name: github.com/kubeflow/model-registry
  dependency-version: 0.2.17
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @cypress/code-coverage from 3.13.12 to 3.14.1 in /clients/ui/frontend (#1096)

Bumps [@cypress/code-coverage](https://github.com/cypress-io/code-coverage) from 3.13.12 to 3.14.1.
- [Release notes](https://github.com/cypress-io/code-coverage/releases)
- [Changelog](https://github.com/cypress-io/code-coverage/blob/master/.releaserc)
- [Commits](https://github.com/cypress-io/code-coverage/compare/v3.13.12...v3.14.1)

---
updated-dependencies:
- dependency-name: "@cypress/code-coverage"
  dependency-version: 3.14.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump dotenv from 16.4.7 to 16.5.0 in /clients/ui/frontend (#1097)

Bumps [dotenv](https://github.com/motdotla/dotenv) from 16.4.7 to 16.5.0.
- [Changelog](https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md)
- [Commits](https://github.com/motdotla/dotenv/compare/v16.4.7...v16.5.0)

---
updated-dependencies:
- dependency-name: dotenv
  dependency-version: 16.5.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump junit-report-merger from 7.0.0 to 7.0.1 in /clients/ui/frontend (#1098)

Bumps [junit-report-merger](https://github.com/bhovhannes/junit-report-merger) from 7.0.0 to 7.0.1.
- [Release notes](https://github.com/bhovhannes/junit-report-merger/releases)
- [Commits](https://github.com/bhovhannes/junit-report-merger/compare/v7.0.0...v7.0.1)

---
updated-dependencies:
- dependency-name: junit-report-merger
  dependency-version: 7.0.1
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump koa and @module-federation/enhanced in /clients/ui/frontend (#1104)

Bumps [koa](https://github.com/koajs/koa) to 2.16.1 and updates ancestor dependency [@module-federation/enhanced](https://github.com/module-federation/core/tree/HEAD/packages/enhanced). These dependencies need to be updated together.


Updates `koa` from 2.15.4 to 2.16.1
- [Release notes](https://github.com/koajs/koa/releases)
- [Changelog](https://github.com/koajs/koa/blob/master/History.md)
- [Commits](https://github.com/koajs/koa/compare/2.15.4...v2.16.1)

Updates `@module-federation/enhanced` from 0.11.1 to 0.13.1
- [Release notes](https://github.com/module-federation/core/releases)
- [Changelog](https://github.com/module-federation/core/blob/main/packages/enhanced/CHANGELOG.md)
- [Commits](https://github.com/module-federation/core/commits/v0.13.1/packages/enhanced)

---
updated-dependencies:
- dependency-name: koa
  dependency-version: 2.16.1
  dependency-type: indirect
- dependency-name: "@module-federation/enhanced"
  dependency-version: 0.13.1
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tarilabs committed:
gha: refresh AppArmor settings for mysql in gha (#1112)

* gha: remove obsolete apparmor settings for mysql in gha

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* align manifest between #267 and #703

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Revert removal (keep AppArmor) and change script

This reverts commit e00cd80246ab99bc2d53b76a25d4bebac5833df3.

trying with replacing with apparmor-utils

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* pin ubuntu runners to fetch apparmor settings correctly

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Revert "pin ubuntu runners to fetch apparmor settings correctly"

This reverts commit 769d4527b7b8eb64a84e6c0b73d872af8e312106.

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* attempt by forcing apt mirror to lookup for AppArmor

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* test by removing directly the profile

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Revert "test by removing directly the profile"

This reverts commit 7eb045518671f33404fa2c10960aa1e7608f41a4.

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* simply remove with apparmor_parser -R

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

* Revert "align manifest between #267 and #703"

This reverts commit 6c1a4c8732aa0e4431989a8e60197bb23fabd9e4.

This is taken care in separate PR:
https://github.com/kubeflow/model-registry/pull/1113

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

---------

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

vikas-saxena02 committed:
sorted out kustomize warnings in overlays/db and overlays/postgres (#1109)

* sorted out kustomize warnings in overlays/db

Signed-off-by: Vikas Saxena <Vikas.Saxena.2006@gmail.com>

removed comments in kustomization.yaml in overlays/db

Signed-off-by: Vikas Saxena <Vikas.Saxena.2006@gmail.com>

sorted out kustomize warnings in overlays/postgres

Signed-off-by: Vikas Saxena <Vikas.Saxena.2006@gmail.com>

disabled autocreation of fields in overlays/db

Signed-off-by: Vikas Saxena <Vikas.Saxena.2006@gmail.com>

* Update manifests/kustomize/overlays/db/kustomization.yaml

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>

* Update manifests/kustomize/overlays/db/kustomization.yaml

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>

* Update manifests/kustomize/overlays/db/kustomization.yaml

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>

* Update manifests/kustomize/overlays/db/kustomization.yaml

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>

* Update manifests/kustomize/overlays/postgres/kustomization.yaml

Co-authored-by: Paul Boyd <paul@camelot.email>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>

---------

Signed-off-by: Vikas Saxena <Vikas.Saxena.2006@gmail.com>
Signed-off-by: Vikas Saxena <90456542+vikas-saxena02@users.noreply.github.com>
Co-authored-by: Paul Boyd <paul@camelot.email>

pboyd committed:
chore(manifests): remove default namespace (#1074)

Remove `namespace: kubeflow` from the core model registry manifests.

Fixes #1045

Signed-off-by: Paul Boyd <pboyd@redhat.com>

tarilabs committed:
chore: move 1st time contrib GHA in Beta mode (#1088)

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

tarilabs committed:
docs: document ROADMAP.md per current process (#1110)

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

Al-Pragliola committed:
Merge remote-tracking branch 'origin/main' into al-pragliola-20250516-sync

tarilabs reviewed (approved):
thanks a lot @Al-Pragliola 

/lgtm
/approve
